src/clusterfuzz/_internal/system/archive.py

```diff
@@ -123,52 +140,54 @@ def close(self) -> None:
     raise NotImplementedError
 
   @abc.abstractmethod
-  def extractall(self, path, members=None, trusted=False) -> None:
+  def extract_all(self,
+                  path: Union[str, os.PathLike],
+                  members: List[str] = None,
+                  trusted: bool = False) -> None:
     """Extract the whole archive content or the members listed in `members`.
 
     Args:
-        path (str): the path where the members should be extracted.
-        Defaults to None.
-        members ([str], optional): the member names. Defaults to None.
-        trusted (bool, optional): whether the archive is trusted or not.
+        path: the path where the members should be extracted.
+        members (optional): the member names. Defaults to None.
```

Comment:
The fact that these are optional, and their default value, is already documented in the function signature since they are kwargs.